### DISCUSSION TITLE
Question: Is it possible to have a floating model?

### DISCUSSION THREAD

**Discussion by HassanZahirnia** [google/model-viewer#1872] [Q&A]
Hi,

First of all, amazing work! So thanks for that. 
I have a question in mind. Is it possible to put some distance between the model and the ground? Let's say the model is a lamp, or a bird and I want to see it above me.

Is this even technically possible in the AR world? If yes, does this web component supports it or it has to yet be implemented?

**Comment by cdata** [comment]
Hey, sorry for the delayed reply @xperator . It is currently not possible to configure the model to be floating. We have speculated that this would be a useful feature for `<model-viewer>` once we have stable WebXR.

You are able to simulate floating by including invisible nodes in your model. Both `<model-viewer>` and Scene Viewer will include empty nodes when computing the bounding region of the model, so if you have an empty node below the model then the model will appear to be floating when placed in AR.

**Comment by HassanZahirnia** [comment]
@cdata Thanks a lot for the response.
I'm a novice user in 3D world. Could you please explain a bit more about the "invisible nodes" you mentioned?
Is that a `<model-viewer>` feature or you're talking about the 3D modeling side. I use Blender to do my 3D stuff, but I don't know anything about adding or including "invisible nodes" in a model. If you could give me some pointers that would be great.

Also, how far can I elevate the model? Is there a size limit or any sort of container box that would prevent the model from growing too large or too distant from the ground?
